strawberry/django/views.py

```diff
@@ -62,10 +64,19 @@ def dispatch(self, request, *args, **kwargs):
         )
 
         response_data = {"data": result.data}
-
         if result.errors:
             response_data["errors"] = [
                 format_graphql_error(err) for err in result.errors
             ]
+            self.log_errors(result.errors)
 
         return JsonResponse(response_data)
+
+    def log_errors(self, errors):
+        if not self.logger:
+            return
+        for error in errors:
+            if hasattr(error, "original_error") and error.original_error:
+                self.logger.error(error.original_error)
+            else:
+                self.logger.error(error)
```

Comment:
sorry @estyxx, I completely forgot to reply to this. I was thinking about how to make this different, as right I'm not sure this is ideal. Will thinking about it over the next few days 😊